go.mod

```diff
@@ -1,14 +1,126 @@
 module github.com/decred/decrediton
 
-go 1.16
+go 1.18
```

Comment:
Cannot build against dcrdex without updating to at least go 1.17

https://github.com/decred/decrediton/pull/3791/checks#step:5:192